numpy/core/src/umath/ufunc_object.c

```diff
@@ -5196,6 +5199,8 @@ PyUFunc_RegisterLoopForDescr(PyUFuncObject *ufunc,
                 current->nargs = ufunc->nargs;
             }
             else {
+                PyErr_SetString(PyExc_ValueError,
+                    "loop already registered");
```

Comment:
Shouldn't this be a RuntimeError (although considering the discussion, it is good to know it didn't get replaced)?

Reply:
ok, changing